.github/workflows/build_linux.yml

```diff
@@ -13,78 +13,80 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - job-name: "focal gcc7"
-            os-version: "20.04"
-            c-compiler: "gcc-7"
-            cxx-compiler: "g++-7"
-            use-syslibs: false
-            shared-libscsynth: false
 
-          - job-name: "focal gcc9 shared libscsynth"
-            os-version: "20.04"
+          - job-name: ""
```

Comment:
why is the jobname now empty? I thought it would be `22.04 gcc9 shared libscsynth` in this case.

Reply:
I made a change so that `job-name` really becomes "job name suffix": the name is now constructed as [`Ubuntu ${{ matrix.os-version }} ${{ matrix.c-compiler }} ${{ matrix.job-name }}`](https://github.com/supercollider/supercollider/pull/6457/files#diff-857070821eb9baccd26d4c48bc577d65473e8e4f0dfe3a69473fe0f26bcfcbc2R89) - less duplicated typing.

It's hard to see in the diff, but the `use system libraries` and `share libscsynth` were [moved to the `gcc-13` and `gcc-14` builds](https://github.com/supercollider/supercollider/blob/cb15f22eb62dbd137441ce7e7974ad89714c06f0/.github/workflows/build_linux.yml#L39-L51), respectively.

Reply:
Oh right, should have looked closer, sorry!